packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -318,7 +318,14 @@ export const Table = memo((props: Props) => {
   };
 
   return (
-    <div {...getTableProps()} className={tableStyles.table} aria-label={ariaLabel} role="table" ref={tableDivRef}>
+    <div
+      {...getTableProps()}
+      className={tableStyles.table}
+      aria-label={ariaLabel}
+      role="table"
+      ref={tableDivRef}
+      style={{ width, height }}
```

Comment:
this just adds width and height to the wrapping div for the table. It should never extend these bounds anyway. And it makes it easier to use Table inside AutoSizer without wrapping it in a div which is otherwise needed